### ISSUE TITLE
dotnet pack/fix-nuspecs is still slow

### ISSUE THREAD

**Issue by matthid** [fsprojects/Paket#2678]
### Description

We need to ensure that all commands we execute from within msbuild are either one-time or blasing fast. Otherwise the experience with multiple projects is really really bad.

This issue tracks the "dotnet pack" scenario ("dotnet restore/build" was https://github.com/fsprojects/Paket/pull/2675).

- A quick gain is getting rid of `Argu`

I don't know other options yet, we might need to profile `fix-nuspecs`.

**Comment by 0x53A** [comment]
Have you tried NGENing paket? That would eliminate the JIT cost

**Comment by matthid** [comment]
If `Argu` is the bottleneck (what it was in "restore" than I don't think it would yield much. Problem is the reflection used in Argu. But worth a try anyway.

**Comment by 0x53A** [comment]
I don't think you see the JIT cost if you profile the App. Afaik the profiler just attaches after jitting. I would do two wall-time clock measurements, once before and once after NGENing.
